bin/client/main.rs

```diff
@@ -144,7 +144,21 @@ async fn run() -> std::result::Result<(), ClientError> {
             }
             .await
             .map_err(map_storage_error)?;
-            tokio::io::stdout().write_all(&toml::to_vec(&inv)?).await?;
+
+            match info_opts.output {
+                Some(format) if &format == "toml" => {
```

Comment:
non-blocking nit: this could probably be a match on the `Option` first, then a match for the string against string literals for the output.

Reply:
I'll refactor this in a different PR because it's in multiple places